pkg/util/fielderrors/fielderrors.go

```diff
@@ -1,207 +0,0 @@
-/*
```

Comment:
It is way too much of a friday for me to review in depth, but I thought these were here to break some cycle? Maybe not. I like where you put it, though I would consider a sub-package there, too.